packages/react/src/primitives/ToggleButtonGroup/__tests__/ToggleButtonGroup.test.tsx

```diff
@@ -15,7 +15,7 @@ describe('ToggleButtonGroup: ', () => {
     const [value, setValue] = React.useState(['test-button-1']);
     return (
       <ToggleButtonGroup
-        onChange={(value: string[]) => setValue(value)}
+        onChange={(value) => setValue(value as string[])}
```

Comment:
Non-blocking: Kind of think we should just fix the typing of `ToggleButtonGroup`, the `onChange` handler should know the type of param it passes in to `onChange` based on the `value`

Reply:
Yeah, I was wondering about this too. I haven't had much luck with conditional types though